jekyll/_cci2/deploy-android-applications.adoc

```diff
@@ -0,0 +1,333 @@
+---
+categories: [platforms]
+version:
+- Cloud
+- Server v3.x
+---
+= Deploy Android Applications
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Learn to deploy Android apps through CircleCI. 
+:icons: font
+:toc: macro
+:toc-title:
+
+Overview
+
+In this tutorial, you will learn how to automatically deploy your Android app through CircleCI with link:https://fastlane.tools/[Fastlane]. 
+
+To complete this tutorial, you will need the following:
+
+- An Android project, built using link:https://gradle.org/[Gradle]. 
+// TODO: Check Gradle prereq.
+// TODO: Include link to sample project?
```

Comment:
I would remove these two notes before merging. They won't show up, but if someone inspected the page they'd be able to see them and it's not the best look.